src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,103 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import React, { useContext } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../types';
+import { PeopleFilteredContext } from '../contsxts/PeopleFilteredContext';
```

Comment:
The import path for `PeopleFilteredContext` is incorrect. It should be '../contexts/PeopleFilteredContext' instead of '../contsxts/PeopleFilteredContext'.